test/edu-benefits/components/RotcHistoryFields.unit.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import RotcHistoryFields from '../../../src/js/edu-benefits/components/RotcHistoryFields';
+import { createVeteran } from '../../../src/js/edu-benefits/utils/veteran';
+
+describe('<RotcHistoryFields>', () => {
```

Comment:
That seems fine to me.  I thought I might test some of the `GrowableTable` functionality in here, but then I realized that it wasn't really part of this PR.

Removing for now.